DependencyInjection/CmfSeoExtension.php

```diff
@@ -216,4 +230,16 @@ private function loadErrorHandling($config, ContainerBuilder $container)
             }
         }
     }
+
+    private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBuilder $container)
+    {
+        if ($config['enabled']) {
```

Comment:
i think you don't need the check here, its already checked to see if loadSitemapHandling needs to be called.